Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx

```diff
@@ -253,16 +267,34 @@ void ITSMFTDeadMapBuilder::PrepareOutputCcdb(DataAllocator& output)
 
   info.setAdjustableEOV();
 
-  LOG(info) << "Sending object " << info.getPath() << "/" << info.getFileName()
-            << " of size " << image->size() << "bytes, valid for "
-            << info.getStartValidityTimestamp() << " : " << info.getEndValidityTimestamp();
+  if (ccdburl.empty()) {
```

Comment:
@nicolovalle should this if be `if(ec != nullptr)` ? I mean:
- I imagine the `ccdburl` always set to  `http://o2-ccdb.internal`
- If the EoS exists, `endOfStream` is called and we ship to ccdb-populator since `ec` is NOT `nullptr`
- If the EoS does not exist, `stop` is called and we ship directly to ccdb-populator since `ec` is `nullptr`

Reply:
Hi @iravasen  yes it makes sense to check that `ec` is not `nullptr`, but I would not set `ccdburl = o2-ccdb.internal` as default so that one has more control in case the workflow is used offline. 
In the new commit I request either `ec != nullptr` or `ccdburl not empty`.

Reply:
@iravasen Sorry, I may have overlooked your message. 
Do you mean that the function "endOfStream(...)" is always called even if the EoS does not exist? If that's the case, I don't even need to implement `stop()`. 
At the moment it is implemented assuming that:
- `endOfStream()` is called only if EoS is received --> `prepareOutputCccd` is called with a not null pointer. Here it's impossible to enter the local push condition.
- `stop()` is always called, but it does the local push only if `endOfStream` has not been processed before.

Reply:
with the first bullet I simply meant to put `--ccdb-url "http://o2-ccdb.internal"` at P2 so that the workflow has always the possibility to ship in `stop()` in case of issues with EoS.

Reply:
Ah yes, understood, that’s done in the .sh.

Reply:
thanks! All good to me, sorry for the late reply.